README.md

```diff
@@ -626,6 +627,31 @@ Below are the steps to create a basic data architectureℹ️ of a SQL server an
 
 > ℹ️ In larger organisations, a data engineer may be less likely to create resources for live production use and instead, it would be the responsibility of more specialised roles such as DevOps engineers or cloud architects. These professionals often have deeper expertise in managing production environments, ensuring high availability, scalability, and compliance with regulatory requirements. However, it is still a useful skill to build resources in development environments, but remember, these are not suited for production due to security risks. For production, stringent security measures are necessary to protect sensitive data and comply with industry standards. This includes implementing strong firewall rules, data encryption (both in transit and at rest), and regular access and activity audits. For a detailed guide on best practices, particularly for a production setup, refer to: [Azure SQL Database Security Best Practices](https://learn.microsoft.com/en-gb/azure/azure-sql/database/security-best-practice?view=azuresql).
 
+<a id="azureVM"></a>
+
+### 💻9️⃣ Setup a windows server VM in your Azure sandbox
+
+Due to the way that the organisation's networking is setup it is not possible to connect directly to SQL servers outside of the corporate network. If using a work computer, you need to setup a virtual machine to install the tools in the next section on. That way the instructions in [section 10](#tools) will work as written.
```

Comment:
I think it would be useful to give a brief explanation of why this is needed. Suggest adding: ", as the connection is being blocked by the companies firewall" after ... the corporate network.

Then at the end suggest adding: "The virtual machine that is being setup in the instructions below is a server that is running the windows operating system. you will then be able to logon to this server and use it in a similar way as you would use your laptop. It will be setup with the windows operating system so you can install any windows compatible software, e.g VS code, SQL Server Manager and Azure Data Studio. You will also have administrator rights so you can perform the installation yourself. This server doesn't have the firewall restrictions that your laptop has, so it is able to connect to the database that is created in the previous section. The instructions below guide you through how to setup the server and then how to remotely access the server"

Reply:
Added recommended updates